addons/mail/static/src/views/web/form/email_template_form_controller.js

```diff
@@ -0,0 +1,73 @@
+import { ConfirmationDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
+import { escape } from "@web/core/utils/strings";
+import { FormController } from "@web/views/form/form_controller";
+import { HistoryDialog } from "@html_editor/components/history_dialog/history_dialog";
+import { markup } from "@odoo/owl";
+import { _t } from "@web/core/l10n/translation";
+import { useService } from "@web/core/utils/hooks";
+
+export class EmailTemplateFormController extends FormController {
+    setup() {
+        super.setup();
+        this.notifications = useService("notification");
+    }
+
+    /**
+     * @override
+     */
+    getStaticActionMenuItems() {
+        return {
+            ...super.getStaticActionMenuItems(),
+            openHistoryDialog: {
+                sequence: 15,
+                icon: "fa fa-history",
+                description: _t("Version History"),
+                callback: () => this.openHistoryDialog(),
```

Comment:
Alternative notation
```suggestion
                callback: this.openHistoryDialog.bind(this),
```